packages/core/src/compiler/styleTypeChecker/styleTypeChecker.ts

```diff
@@ -0,0 +1,102 @@
+import { Shape, ShapeType } from "../../shapes/Shapes";
```

Comment:
I think it's okay to just put this code into `Style.ts` or at least under of `compiler` folder. We don't really do much folder nesting in the repo and I want to keep the directory structure simple.

Reply:
I'll put it under the `compiler` folder.

Reply:
Done